### ISSUE TITLE
jax.numpy.nanpercentile with axis as tuple

### ISSUE THREAD

**Issue by clee1994** [google/jax#9277]
currently, the following code results in a NotImplementedError:
```
jax.numpy.nanpercentile(jnp.ones((3,3,12,24)), 99.9, axis=(0,1,2))
```

```
 File "/home/clemens/.local/lib/python3.8/site-packages/jax/_src/numpy/lax_numpy.py", line 6359, in _quantile                                       
    raise NotImplementedError("Tuple values for axis are not implemented")   
```
However the docstring actually explicitly mentions tuple as a valid option for axis: https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.nanpercentile.html

same for jax.numpy.percentile.html

**Comment by jakevdp** [comment]
Thanks for the report – this feature hasn't been implemented yet. Regarding the docstring, the bulk of the content is copied directly from the corresponding numpy function (it's easy to miss, but at the top you'll see something like "original docstring below") because of that, the details are not always accurate, particularly when corner cases are explicitly unimplemented.

We can leave this issue open to track implementing this feature.

**Comment by MichaelMarien** [comment]
I'd like to give this one a try. Might take some time though as I'm quite new Jax. If that's no problem, feel free to assign it to me!

**Comment by clee1994** [comment]
Thank your so much @MichaelMarien @jakevdp that was a very quick fix :)
